src/ImageRunModal.jsx

```diff
@@ -352,7 +352,7 @@ export class ImageRunModal extends React.Component {
                                    type='number'
                                    value={dialogValues.cpuShares}
                                    step={1}
-                                   min={0}
+                                   min={2}
```

Comment:
2 seems so arbitrary.. If we allow a container to have that little CPU share (0.2% of the CPU), then we may just as well cap it to 1. Unless we keep 0 as "default value", aka. "don't explicitly set it"?

Reply:
`cpu-shares` is relative weight. So I you set up one with `2` and the other one with `4`, it is the same as `512` and `1024`.
```To modify the proportion from the default of 1024, use the --cpu-shares flag to set the weighting to 2 or higher.```

Reply:
I  see -- I thought it was not just between containers, but between *all* processes in the system. If it's the former, how does the collective set of containers get prioritized to non-container processes?

Reply:
from `man podman-run`:

> By  default,  all  containers  get the same proportion of CPU cycles. This proportion can be modified by changing the container's CPU share weighting **relative to the weighting of all other running containers**.



> how does the collective set of containers get prioritized to non-container processes?

I am not the right person to respond to that

Reply:
With 

    cat /dev/urandom >/dev/null
    sudo podman run --cpu-shares=1024 -it --rm busybox sh -ec 'cat /dev/urandom >/dev/null'

the container cat process uses 25% total CPU (i. e. one of the four cores), in other words, it gets the maximum CPU amount possible. It's excactly the same with `--cpu-shares=2`, so it seems this is indeed only relative to other containers. But two podman run instances with shares 2 and 10 respectively, still both cats get 100% on their individual CPUs, and use up 50% of the CPU in total, though.

So I'm not sure how to even demonstrate the effect of that..

Reply:
Anyway, closing this thread as the "2" is documented in the podman docs.